common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -85,4 +85,14 @@ trait ABTestSwitches {
     sellByDate = new LocalDate(2019, 5, 31),
     exposeClientSide = true
   )
+
+  Switch(
+    ABTests,
+    "ab-commercial-us-mobile-sticky",
+    "Test whether changes to Consent Banner increases proportion of non-EU users who interact with it",
```

Comment:
Is this description copied from an irrelevant test?

Reply:
It is. Good catch. Thanks.